.changeset/config.json

```diff
@@ -5,6 +5,7 @@
   "linked": [],
   "access": "public",
   "baseBranch": "main",
+  "skipCI": false,
```

Comment:
Do we need to set "commit: true" as well: https://github.com/changesets/changesets/blob/c68536edf4c04e7fdf5594ec9c69471cd86fd0ce/__fixtures__/linked-packages-with-linked-dev-dep/.changeset/config.js#L66